docs/content/guides/dagster/re-execution.mdx

```diff
@@ -0,0 +1,159 @@
+
+\*\*\*\* INSERT NEW SCREENSHOT OF UNREABLE PIPELINE \*\*\*\*
+
+<Image
+alt="Dagit Re-execution"
+src="/images/concepts/dagit/re-execution.png"
+width={3808}
+height={2414}
+/>
+
+### Re-execution Options
+
+There are several options for launching a re-execution. No matter which one you choose, the re-executed run is linked to the original run.
+
+- **All Steps**: Re-execute the pipeline run from scratch. This option is most relevant upon testing pipeline runs end to end, if you'd like to associate different runs together.
+- **Selected Step**: Re-execute the selected step(s). This can be done independent of the step status. Your pipeline is likely large, and you know exactly which steps to execute. This can be done by clicking on the boxes in the gantt chart view.
+- **From Selected**: Re-execute the pipeline downstream from the selected steps. If a particular step fails, this option is most relevant if your intent is to run all downstream steps regardless of status. You're likely just developing a single step, and want to make sure downstream steps work as expected.
+- **From Failure**: Re-execute the pipeline run, skipping steps completed successfully. This is only enabled when the pipeline has failed. You have likely fixed the failed step, and want to run the step and all downstream dependencies again. Dagster will figure out the dependencies for you!
+
+In the above example, re-executing from failure would make sense as the failed task has a 50% chance of succeeding on the next run.
+
+### Selecting Steps
+
+Within Dagit, a single or multiple steps may be selected simply by clicking them with the mouse. Alternatively, the text-box can be leveraged by typing step names to be re-run, adding a `+` for every subsequent step to be selected. This is demoed in the video below:
+
+<iframe
+  width="560"
+  height="315"
+  src="https://www.youtube.com/embed/LWc8MxSN22c"
+  frameBorder="0"
+  allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+  allowfullscreen
+></iframe>
+
+## Relevant APIs
+
+Re-execution can be triggered via the API as well.
+
+| Name                                                                                       | Description                         |
+| ------------------------------------------------------------------------------------------ | ----------------------------------- |
+| <PyObject module="dagster" object="reexecute_pipeline" displayText="reexecute_pipeline" /> | Reexecute an existing pipeline run. |
+
+Consider the pipeline `unreliable_pipeline`, which has a solid named `unreliable`.
+
+```python file=../../reexecution/reexecution_api.py
+from dagster import DagsterInstance, execute_pipeline, reexecute_pipeline
+from reexecution.pipeline.unreliable_pipeline import unreliable_pipeline
+
+
+def reexecution_api_example():
+    instance = DagsterInstance.ephemeral()
+    run_config = {"intermediate_storage": {"filesystem": {}}}
```

Comment:
this is our bad that we didn't communicate the deprecation well. the recommended way is to use `fs_io_manager` as mentioned above and you don't need to specify this in run_config because we've set a filesystem io manager at definition time. so instead you can simply do:
```
pipeline_result_full = execute_pipeline(unreliable_pipeline, instance=instance)
```

Reply:
Got it, updating